<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="sun">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>150</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>sun</string>
    </property>
   </widget>
   <widget class="QLabel" name="red">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>red</string>
    </property>
   </widget>
   <widget class="QSlider" name="LM35_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>21</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>10</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>21</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>5</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LDR">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>580</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LDR</string>
    </property>
   </widget>
   <widget class="QLabel" name="yellow">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>220</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>yellow</string>
    </property>
   </widget>
   <widget class="QLabel" name="LDRvalue">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>390</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LDRvalue</string>
    </property>
   </widget>
   <widget class="QDial" name="LDR_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>261</width>
      <height>311</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>1023</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="green">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>300</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>81</height>
     </size>
    </property>
    <property name="text">
     <string>green</string>
    </property>
   </widget>
   <widget class="QLabel" name="motion">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>220</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>motion</string>
    </property>
   </widget>
   <widget class="QLabel" name="LM35value">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LM35value</string>
    </property>
   </widget>
   <widget class="QLabel" name="LM35">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>580</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LM35</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CheckBox">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>460</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <zorder>sun</zorder>
   <zorder>red</zorder>
   <zorder>LM35_2</zorder>
   <zorder>LDR</zorder>
   <zorder>yellow</zorder>
   <zorder>green</zorder>
   <zorder>motion</zorder>
   <zorder>LM35value</zorder>
   <zorder>LM35</zorder>
   <zorder>LDR_2</zorder>
   <zorder>LDRvalue</zorder>
   <zorder>CheckBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
